openedx_events/event_bus/avro/tests/test_deserializer.py

```diff
@@ -102,6 +102,18 @@ def test_default_coursekey_deserialization(self):
         self.assertIsInstance(course_deserialized, CourseKey)
         self.assertEqual(course_deserialized, course_key)
 
+    def test_default_usagekey_deserialization(self):
```

Comment:
The missing docstring is a bit concerning, but I understand you're following the pattern in this test file. So that gives me a bit of work for the future.

Reply:
@mariajgrimaldi Started by adding docstrings in the tests that I created. :)